Avoid w32api DATADIR definition, which conflicts with internal usage:

<command-line>:0:9: error: expected identifier or ‘(’ before string constant

--- origsrc/gtksourceview-3.22.2/gtksourceview/gtksourceview-init.c	2016-11-16 08:57:46.000000000 -0600
+++ src/gtksourceview-3.22.2/gtksourceview/gtksourceview-init.c	2017-03-08 12:29:21.369922600 -0600
@@ -37,6 +37,7 @@
 #include "gtksourcestyleschememanager.h"
 
 #ifdef G_OS_WIN32
+#define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 
 static HMODULE gtksourceview_dll;
